app/views/content/sandbox.md

```diff
@@ -0,0 +1,27 @@
+Our [Sandbox environment](https://sandbox.apply-for-teacher-training.education.gov.uk) is a separate version of our service that you can be use to test.
+
+## Things that you can do on the Sandbox
+
+- Create applications as a candidate
+- Manage applications in the provider section
+- Test [API integrations](/api-docs)
+
+Find it at:
+
+[sandbox.apply-for-teacher-training.education.gov.uk](https://sandbox.apply-for-teacher-training.education.gov.uk)
+
+## Warning
+
+- Only use your **own email address** to avoid emailing real people
+- Do not enter real personal data into this system - use fake names, addresses and other information
+
+## Differences with the real version
+
+To make testing easier, the sandbox differs from the real version:
+
+- You can generate test applications on the Sandbox
+- Applications are normally sent to the provider 5 days after submission, to allow candidates to edit the form. In the sandbox applications are sent immediately.
+- In the service, applications are sent to the provider only when 2 references have been provided. You can use a dummy email addresses that will automatically fill in the reference feedback.
```

Comment:
It makes sense to specify those dummy email addresses here. As things stand they are `refbot1@example.com` and `refbot2@example.com`